docs/guides/miscellaneous/migration_guide.md

```diff
@@ -155,15 +155,25 @@ Now you are ready to manually migrate Datasources and Checkpoints to be compatib
 ### Manually migrate Datasources from V2 to V3
 
 The first manual step needed is to convert the V2-style Datasource to a V3-style one. The following documentation 
-contains examples for data read-in using `pandas` and `spark`. 
+contains examples for data read-in using `pandas`, `spark`, and a database, using `postgresql` as an example. 
+
+:::tip
+
+The configurations for `pandas`, `spark` and `postgresql` shown in this guide are available as part of the `great-expectations` repository. 
+Please feel free to use the complete-and-working configurations found [here](https://github.com/great-expectations/great_expectations/tree/develop/tests/test_fixtures/configuration_for_testing_v2_v3_migration)
+to help with your migration. 
+
+:::
 
 <Tabs
   groupId="configurations-pandas-spark-postgres"
   defaultValue='pandas'
   values={[
   {label: 'Pandas', value:'pandas'},
   {label: 'Spark', value:'spark'},
+  {label: 'Database', value:'db'},
```

Comment:
Currently the tab says `Database`. Should it be `Postgresql` instead?

Reply:
I think Database is fine since you're using Postgres as a way to explain Database configuration

Reply:
You can maybe point to the other DB guides at the end of the Postgres section if you want to make it general?